client/src/components/StudentProfile.js

```diff
@@ -39,7 +39,7 @@ const StudentProfile = () => {
 					{student.previousFeedback}</p>
 			</div>
 
-			<FeedbackForm />
+			<FeedbackForm student={student} />
```

Comment:
How are we passing props here while we don't have any props in the function definition on line 5?

Reply:
No need for student. not sure why it is there. You are right, we can remove it.